### PR TITLE
Room UI Redesign: Miscellaneous Fixes

### PR DESCRIPTION
**Pull request by robertlong**
- Center the hero content on mobile
- Fixes chat message line height
- Fixes the modal width for the verify auth page and all other modals on mobile
- Fixes https://github.com/mozilla/hubs/issues/3702
- Fixes https://github.com/mozilla/hubs/issues/3694

### CODE REVIEW

**Comment by robertlong** [comment]
Couldn't test the chat message spawning locally so I tested on staging. There seems to be some issues with the new presence-log styles. Fixing now.

**Comment by robertlong** [comment]
I couldn't find the right SVG foreign element CSS incantation so I fixed the chat message rendering with html2canvas which needs thorough testing, particularly with VR perf. I could have rendered chat bubbles manually, but multi-line text, emojis etc. would have been a lot of work. If perf isn't an issue I think html2canvas is easier to maintain as we can more or less style chat messages as we want. It could also be used for more things in the future.

File: src/react-components/home/HomePage.scss [hunk 1]
```diff
@@ -37,6 +37,7 @@
   display: flex;
   flex-direction: column;
   align-items: center;
+  flex: 1;
```
Comment (johnshaughnessy) [inline src/react-components/home/HomePage.scss:40]: :muscle:

File: src/utils/chat-message.js [hunk 2]
```diff
@@ -4,57 +4,33 @@ import { toArray as toEmojis } from "react-emoji-render";
...
 
-  const cleanedBody = (monospace ? body.substring(1, body.length - 1) : body).trim();
+  const multiline = messages.length > 1;
 
-  const bodyWithEmojis = toEmojis(cleanedBody);
+  let emoji = false;
 
-  const formattedBody = <Linkify properties={{ target: "_blank", rel: "noopener referrer" }}>{bodyWithEmojis}</Linkify>;
+  if (messages.length === 1) {
+    const emojiComponents = toEmojis(cleanedBody);
```
Comment (johnshaughnessy) [inline src/utils/chat-message.js:25]: i am a potato
here are some emojis

File: src/react-components/chat-message.js [hunk 3]
```diff
@@ -4,12 +4,10 @@ import PropTypes from "prop-types";
 import styles from "../assets/stylesheets/presence-log.scss";
 import classNames from "classnames";
 import { toArray as toEmojis } from "react-emoji-render";
-import serializeElement from "../utils/serialize-element";
+import html2canvas from "html2canvas";
```
Comment (johnshaughnessy) [inline src/react-components/chat-message.js:6]: I don't have a VR headset handy to test perf... (you mentioned in your comments might be a problem)

But it seems good that `html2canvas` only pulls in a single dependency.

**Review by johnshaughnessy — APPROVED** [review]

File: src/utils/chat-message.js (hunk 2 above)
Comment (robertlong) [reply src/utils/chat-message.js:25]: :open_hands: :potato:

**Pull request merged**